platform/podjs/src/browserPod.ts

```diff
@@ -59,6 +77,10 @@ class LocalStoragePolyIn implements PolyIn {
         });
     }
 
+    /**
+     * It adds the given quads to the store
```

Comment:
Maybe precise that in this case it's using local storage. It's important, because it's got a series of limitations in size and so on.